### DISCUSSION TITLE
Icons on Map

### DISCUSSION THREAD

**Discussion by MeusRex** [MeusRex/pf2e-chex#19] [Ideas]
For some Features/Improvements/Resources it would be nice to show an icon on the map. 
For example Towns, Villages or a known cave entrance. 

It would be nice to configure:
Image or FA Icon
Tint Color
Priority (To prevent cluttering only the most important 1? 2? 3? should show. Mabye with a star symbol that there is more.)

This will make finding a hex of interest way easier than hovering over each.

**Comment by federicoacker** [comment]
I've actually been experimenting with this one, I have tiles configured with features and things like that and they are quite hard to find, maybe another solution would be to add another overlay level for features? so we'd have "terrain overlay", "realm overlay" and then "feature overlay" that just colors any tile that has at least one feature so they're easy to find and check for the dm, maybe make it so that if the features are hidden then the color doesn't show up for players
